### ISSUE TITLE
WCAG2.2: 3.2.7 Visible Controls (AA) - Confirmation on navigation menus

### ISSUE THREAD

**Issue by emleml** [w3c/wcag#1845]
Hi. For proposed **3.2.7 Visible Controls**, I have a question to confirm something. I am aware this is not best accessibility practice in general due to touch screen devices not experiencing this pattern how intended, and considerations needing to be taken for Content on Hover of Focus, however, imagine a situation:

I have a top level menu of navigation links where on hover or focus a dropdown menu appears showing sub-menu links. Each link, including the top-level link is an anchor tag with a href. On hover and focus I can access all links. On tap/on click I cannot access the dropdown menu but the top level link will take me to a "splash page" where all sub menu links are visible and available as a list in the same relative order as they were displayed on hover/focus.

Would this be a non-compliance against Visible Controls? The success criterion requires the visible information to be displayed on the same page or within the same process (example given, opening an email to see persistent controls), however I am not sure whether navigation is categorised as part of a process.

Does this count as "information needed to identify that user interface component"?

I would counteract if it is a non-compliance, that as a web-author I could decide to not have a drop down menu at all and instead just use a splash-navigation as my default navigational method.

Ideally the drop down menu would be modified to replace the high level link as a button, and the top level link will be presented as one of the sub-menu items, or an additional control within the top level link acting as the expanding toggle. However, this example is for those websites that are currently using the on hover/on focus design pattern, not that I am suggesting we should use it.

Appreciate any clarification. I see this pattern used a lot across e-commerce websites with mega menus. Whilst generally accessibility often isn't considered for these, for those who have implemented an accessible solution for keyboard users and provided a splash page, would this new criteria cause any concerns?

Thanks all

**Edit:**  Am I correct in that providing a visual indicator, such as a down chevron would be sufficient to pass this SC for the above scenario?

**Comment by mraccess77** [comment]
Yes, it is my understanding that providing a drop down arrow or ... type of indicator would be sufficient to meet the criteria.

**Comment by patrickhlauke** [comment]
a slightly tricky situation, because if the same functionality is also available in a different way (being able to click the dropdown trigger element to be taken to a separate intermediate page that has all the same links as the dropdown), you could even consider the dropdowns an additional "nice to have" functionality. so for a user not to know/remember that certain elements show a dropdown is not a hindrance for them in eventually getting to where they want to get. i'm leaning towards this passing the SC (or rather, the requirement of the SC not applying to this particular pattern)

**Comment by alastc** [comment]
Response above agreed by the working group.

**Comment by bruce-usab** [comment]
@emleml -- just as an FYI, it used to be pretty common for a top-level menu item to be linked to a separate navigation page listing choices from under that top-level menu item (even when those choices could be selected directly via mouse).  At first, this was not
